nav2_behavior_tree/include/nav2_behavior_tree/bt_conversions.hpp

```diff
@@ -59,6 +60,26 @@ inline geometry_msgs::msg::Quaternion convertFromString(const StringView key)
   }
 }
 
+template<>
+inline geometry_msgs::msg::PoseStamped convertFromString(const StringView key)
```

Comment:
Should we not also include the header / frame / timestamp?

Reply:
Does `include the header / frame / timestamp` mean
```
<PoseStampedPort test="now;map;1.0;2.0;3.0;4.0;5.0;6.0;7.0" />
```
?

Or should we consider a different format? For example
```
<PoseStampedPort test="stamp=now, frame_id=map, position=[1.0, 2.0, 3.0], orientation=[4.0, 5.0, 6.0, 7.0]" />
```

Reply:
It means that it should describe the full message. Look at the PoseStamped message and conversions should convert the full message since it could be used in other contexts requiring the header information.

Reply:
Following the same conventions as before is fine, just make sure to describe everything

Reply:
Updated.